api/XRLightProbe.json

```diff
@@ -48,55 +48,6 @@
           "deprecated": false
         }
       },
-      "onreflectionchange": {
-        "__compat": {
-          "spec_url": "https://immersive-web.github.io/lighting-estimation/#dom-xrlightprobe-onreflectionchange",
```

Comment:
@Elchi3 If we're going to have a single entry for both the event and its handler, should the spec here moved down to reflectionchange_event ? (i.e. it has an array of specs)?

Reply:
Yes, I think this is a great idea!